src/vs/workbench/browser/parts/editor/sideBySideEditor.ts

```diff
@@ -31,6 +50,11 @@ export class SideBySideEditor extends BaseEditor {
 	private detailsEditorContainer: HTMLElement;
 
 	private splitview: SplitView;
+	private dimension: DOM.Dimension = new DOM.Dimension(0, 0);
+
+	private onDidCreateEditors = this._register(new Emitter<{ width: number; height: number; }>());
```

Comment:
@joaomoreno the purpose of this event seems only to be to trigger `onDidSizeConstraintsChange`. Can we avoid it and just fire that event directly?

Reply:
Very similar to the problem above, except that the editors are only set once. But the problem is that they are set after the ctor... so we must employ exactly the same solution as above, because someone might reach into `SideBySideEditor.onDidChange` before they are created.